scripts/check-env.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# Check the environment for dependencies and configuration
+#
+
+# count errors, run script to the end before exiting
+ERRORS=0
+fail() {
+	echo "${*}" > /dev/stderr
+	ERRORS=$[ERRORS+1]
+}
+
+# parse the Golang version, return the digit passed as argument
+# 1.13.9 -> go_version 1 -> 1
+# 1.13.9 -> go_version 2 -> 13
+# 1.13.9 -> go_version 3 -> 9
+go_version() {
+	local v="$(go version | cut -d' ' -f3 | sed 's/^go//')"
+	cut -d'.' -f${1} <<< "${v}"
+}
+
+# Golang needs to be > 1.13
+GO_MAJOR=$(go_version 1)
+GO_MINOR=$(go_version 2)
+if ! [ ${GO_MAJOR} -gt 1 ]
+then
+	if ! ( [ ${GO_MAJOR} -eq 1 ] && [ ${GO_MINOR} -ge 13 ] )
+	then
+		fail "go version needs to be >= 1.13"
```

Comment:
can we just exit here, no need to perform other checks as the go version itself is not compatible

Reply:
It is useful to list all failures at the end. Users can then address multiple issues before trying again.